python/dglke/nn/modules/encoder.py

```diff
@@ -9,8 +9,50 @@ def __init__(self,
                  n_entity,
                  n_relation,
                  init_func,
-                 encoder_name='KGEEncoder',):
+                 encoder_name='KGEEncoder',
+                 score_func='TransE'):
         super(KGEEncoder, self).__init__(encoder_name)
```

Comment:
We do not need  encoder_name in KGEEncoder's arg list.
```suggestion
        encoder_name='KGEEncoder'
        super(KGEEncoder, self).__init__(encoder_name)
```